version: '3'
services:
  {{ odoo.service_name }}:
    image: wso2qa4.ccu.cl:5000/odoo:ccu_pos-$SHA
    container_name: {{ odoo.container_name }}
    user: root
    ports:
      - 8069:8069
      {%- if debug %}
      - 8888:3001
      {%- endif %}
    networks:
      - odoo-pos
    entrypoint: ['/entrypoint.sh', 'odoo', '-d', 'MASTER'] 
    volumes:
      - ./logs:/var/log/odoo

  {{ nginx.service_name }}:     
    container_name: {{ nginx.container_name }}
    restart: unless-stopped
    image: nginx:latest
    networks:
      - odoo-pos
    ports:
      - 80:80
      - 443:443
    volumes:
      - ./nginx/nginx-config/nginx_template.conf:{{ nginx.path_config }}
      - ./nginx/ssl/private.key:{{ nginx.path_ssl_certificate_key }}
      - ./nginx/ssl/public.cer:{{ nginx.path_ssl_certificate }}

networks:
  odoo-pos:
      external: true
